### DISCUSSION TITLE
Welcome to PyRFC Discussions!

### DISCUSSION THREAD

**Discussion by bsrdjan** [SAP/PyRFC#230] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amarvin** [comment]
Great idea! I've been a user of `pyrfc` for a few years in my role building AI solutions for optimal business decisions. Often these solutions require data from SAP R/3 Systems, so `pyrfc` is critical. I try to also be engaged in this GitHub project to make `pyrfc` even easier and more useful for future users. Let's build together! 💪
